src/block_cache/block_chunkcache.c

```diff
@@ -288,34 +388,26 @@ __chunkcache_alloc_chunk(WT_SESSION_IMPL *session, wt_off_t offset, wt_off_t siz
      * the allocation function we don't care about the block's size. If more than one chunk is
      * needed to cover the entire block, another function will take care of allocating multiple
      * chunks.
+     *
```

Comment:
Nitpick: the two paragraphs of comments should be separated, "Convert the block offset..." sounds like a continuation of the previous explanation.

Reply:
Updated in [51003f2](https://github.com/wiredtiger/wiredtiger/pull/9759/commits/51003f2c501bd50acf3a6b810714894828425c01)

Reply:
This is a little weird now. I'll push an alternative fix - would like to hear your thoughts on it.

Reply:
Tweaked in b866303 - let me know what you think.